pkgs/by-name/pr/prism-model-checker/package.nix

```diff
@@ -0,0 +1,24 @@
+{
+  lib,
+  stdenv,
+  prism-model-checker-unwrapped,
+  java,
+  symlinkJoin,
+  makeWrapper,
+}:
+symlinkJoin {
```

Comment:
```
├── etc
│   ├── icons
│   │   ├── p128.png -> /nix/store/fp6kyxf5cyxlsqm6mcpxqf777b6gqzxg-prism-model-checker-unwrapped-4.8.1/etc/icons/p128.png
│   │   ├── p16.png -> /nix/store/fp6kyxf5cyxlsqm6mcpxqf777b6gqzxg-prism-model-checker-unwrapped-4.8.1/etc/icons/p16.png
│   │   ├── p24.png -> /nix/store/fp6kyxf5cyxlsqm6mcpxqf777b6gqzxg-prism-model-checker-unwrapped-4.8.1/etc/icons/p24.png
│   │   ├── p256.png -> /nix/store/fp6kyxf5cyxlsqm6mcpxqf777b6gqzxg-prism-model-checker-unwrapped-4.8.1/etc/icons/p256.png
│   │   ├── p32.png -> /nix/store/fp6kyxf5cyxlsqm6mcpxqf777b6gqzxg-prism-model-checker-unwrapped-4.8.1/etc/icons/p32.png
│   │   ├── p48.png -> /nix/store/fp6kyxf5cyxlsqm6mcpxqf777b6gqzxg-prism-model-checker-unwrapped-4.8.1/etc/icons/p48.png
│   │   ├── p64.png -> /nix/store/fp6kyxf5cyxlsqm6mcpxqf777b6gqzxg-prism-model-checker-unwrapped-4.8.1/etc/icons/p64.png
│   │   └── prism.ico -> /nix/store/fp6kyxf5cyxlsqm6mcpxqf777b6gqzxg-prism-model-checker-unwrapped-4.8.1/etc/icons/prism.ico
```
should be in `share/icons`
There are some other files in /etc that should not be there (README.txt, prism.tex, tests, ...)

Also, do we really need the whole `classes` directory? Isn't it included in one of the jars?

Reply:
You are right, I don't think I need the classes folder. Regarding the icons, however,  I assume prism expects them to be in that location, so moving them would require patches.

Reply:
I don't see where the icons are referenced. They can be moved.